service/raft/raft_group0.cc

```diff
@@ -730,6 +730,9 @@ future<> raft_group0::leave_group0() {
 future<> raft_group0::remove_from_group0(gms::inet_address node) {
```

Comment:
Re commit message and PR title: change "regression" to "regression test" - you're not adding a regression, you're adding a regression test.